<ui version="4.0" >
 <class>Editor</class>
 <widget class="QWidget" name="Editor" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>477</width>
    <height>418</height>
   </rect>
  </property>
  <layout class="QGridLayout" >
   <property name="margin" >
    <number>9</number>
   </property>
   <property name="spacing" >
    <number>6</number>
   </property>
   <item rowspan="2" row="0" column="1" >
    <layout class="QHBoxLayout" >
     <property name="margin" >
      <number>0</number>
     </property>
     <property name="spacing" >
      <number>6</number>
     </property>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLabel" name="label" >
         <property name="text" >
          <string>Name:</string>
         </property>
         <property name="buddy" >
          <cstring>fNameEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_2" >
         <property name="text" >
          <string>Account:</string>
         </property>
         <property name="buddy" >
          <cstring>fAccountEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_3" >
         <property name="text" >
          <string>Password:</string>
         </property>
         <property name="buddy" >
          <cstring>fPasswordEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_6" >
         <property name="text" >
          <string>Category</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_4" >
         <property name="text" >
          <string>Last changed:</string>
         </property>
         <property name="buddy" >
          <cstring>fCategoryEdit</cstring>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" >
       <property name="margin" >
        <number>0</number>
       </property>
       <property name="spacing" >
        <number>6</number>
       </property>
       <item>
        <widget class="QLineEdit" name="fNameEdit" />
       </item>
       <item>
        <widget class="QLineEdit" name="fAccountEdit" />
       </item>
       <item>
        <layout class="QHBoxLayout" >
         <property name="margin" >
          <number>0</number>
         </property>
         <property name="spacing" >
          <number>6</number>
         </property>
         <item>
          <widget class="QLineEdit" name="fPasswordEdit" >
           <property name="echoMode" >
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="fPasswordBox" >
           <property name="toolTip" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Show or hide password.&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="whatsThis" >
            <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal; text-decoration:none;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Show or hide password&lt;/p>&lt;/body>&lt;/html></string>
           </property>
           <property name="text" >
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="fCategoryEdit" />
       </item>
       <item>
        <widget class="QDateEdit" name="fDateEdit" />
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="0" >
    <widget class="QComboBox" name="fCategoryFilter" />
   </item>
   <item row="3" column="1" >
    <widget class="QTextEdit" name="fNotesEdit" />
   </item>
   <item row="2" column="1" >
    <widget class="QLabel" name="label_5" >
     <property name="text" >
      <string>Notes:</string>
     </property>
     <property name="buddy" >
      <cstring>fNotesEdit</cstring>
     </property>
    </widget>
   </item>
   <item rowspan="3" row="1" column="0" >
    <widget class="QListView" name="fAccountList" />
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
